framework/db/CheckConstraint.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\db;
+
+/**
+ * CheckConstraint represents the metadata of a table `CHECK` constraint.
```

Comment:
Alternatively, MySQL schema may just throw NotSupportedException since, in fact, CHECK is not supported while syntax is accepted.